js/waves.js

```diff
@@ -446,9 +446,9 @@
                     removeListeners();
                 };
 
-                element.addEventListener('touchmove', touchMove, false);
-                element.addEventListener('touchend', hideEffect, false);
-                element.addEventListener('touchcancel', hideEffect, false);
+                element.addEventListener('touchmove', touchMove, { capture:false, passive: true });
+                element.addEventListener('touchend', hideEffect, { capture:false, passive: true });
+                element.addEventListener('touchcancel', hideEffect, { capture:false, passive: true });
```

Comment:
eh I'm not sure about this. Have a look at https://github.com/Dogfalo/materialize/issues/4720

Reply:
Looks like you're planning on merging passive event receivers already, no? https://github.com/materializecss/materialize/issues/7

Reply:
Isn't this part of waves.js? I guess this is fine because I think the files are completely replaced with the new ones. Correct?

Reply:
> Isn't this part of waves.js? I guess this is fine because I think the files are completely replaced with the new ones. Correct?

hmm the passive event, no it's not https://github.com/fians/Waves/blob/83fc07e2c0a31a10aa5dc2cf97b965113ddd882c/dist/waves.js#L443-L445

Reply:
Ah ok, but I think the 0.8 dev branch is the correct one, the master branch is an old version.

@WaeCo please describe from where you got this file / version so we can check this and please split this into two commits. The first one with the new original files and a reference to where it is from and a second commit with your changes.

Reply:
I already did this, check my the commits.
First commit is code from: https://github.com/fians/Waves/blob/v0.7.6/dist/waves.js

Reply:
Ah ok, that's perfect. I didn't see it on my smartphone. Then it should be fine @Smankusors?

Reply:
Ah ok, that's perfect. Sorry, I didn't see it on my smartphone. Then it should be fine @Smankusors @ChildishGiant?

Reply:
well I think I'm fine... wait didn't you said it will break IE compability? @DanielRuf

Reply:
Not sure what you mean, can you point me to the relevant comment?

Reply:
I guess it's regarding this:

![Screenshot_20200814-151147.jpg](https://user-images.githubusercontent.com/827205/90252884-922e4300-de40-11ea-8799-ef0af744b899.jpg)

![Screenshot_20200814-151155.jpg](https://user-images.githubusercontent.com/827205/90252890-95293380-de40-11ea-85c5-41321b05b08d.jpg)

Reply:
@WaeCo can you change the commit for the passive listeners according to the proposed solution at MDN?

https://developer.mozilla.org/en-US/docs/Web/API/EventTarget/addEventListener#Browser_compatibility

Reply:
Alternatively we can drop the commit for passive event listeners and do it separately and with a global constant for all parts.

Reply:
![grafik](https://user-images.githubusercontent.com/827205/90253221-184a8980-de41-11ea-8835-ed0843d089a4.png)

Reply:
I added it